biomeidentifier.cpp

```diff
@@ -127,7 +128,7 @@ QColor BiomeInfo::getBiomeGrassColor( QColor blockcolor, int elevation )
     // perlin noise generator omitted due to performance reasons
     // otherwise the random temperature distribution selects
     // (below -0.1°C) ‭4C.76.3C‬ or ‭6A.70.39 (above -0.1°C)
-    colorizer = QColor::QColor(0x6a,0x70,0x39);  // hard wired
+    colorizer = QColor(0x6a,0x70,0x39);  // hard wired
```

Comment:
Consider using `QColor::fromRgb(...)` to explicitly show what color system is used. This goes for the other similar changes, too.